api/src/main/java/org/openmrs/api/context/Context.java

```diff
@@ -890,6 +892,57 @@ public static void evictFromSession(Object obj) {
 		getContextDAO().evictFromSession(obj);
 	}
 
+	/**
+	 * Evicts the entity data for a particular entity instance.
+	 *
+	 * @param sessionFactory
+	 * @param entityClass entity class of the instance to evict from the DB cache
+	 * @param object entity instance to evict from the DB cache
+	 */
+	public static void evictSingleEntity(SessionFactory sessionFactory, Class<?> entityClass, OpenmrsObject object) {
+		
+		String uuid = object != null ? object.getUuid() : null;
```

Comment:
This is a weird thing to do... It makes sense in the REST module, which largely operates on UUIDs, but not here.